auth/roles.yaml

```diff
@@ -100,11 +100,10 @@
         - get
```

Comment:
Removed old volumes resource.
Admins will need permissions to read/delete the cache api.